scheduler/duration_based_host_allocator.go

```diff
@@ -492,10 +494,37 @@ func (self *DurationBasedHostAllocator) numNewHostsForDistro(
 		grip.Error(err)
 		return 0, nil
 	}
+
+	underWaterAlert := message.Fields{
+		"distro":    distro.Id,
+		"runtime":   scheduledTasksDuration + runningTasksDuration,
+		"runner":    RunnerName,
+		"message":   "distro underwater",
+		"num_hosts": existingDistroHosts,
+	}
+
 	if !can {
+		underWaterAlert["action"] = []string{
+			"reduce workload;",
+			"add additional hosts to pool;",
+			"deactivate tasks;",
+		}
+		grip.AlertWhen(time.Duration(distroData.totalTasksDuration/float64(distro.PoolSize)) > 7*24*time.Hour,
+			underWaterAlert)
+
 		return 0, nil
 	}
 
+	underWaterAlert["max_hosts"] = distro.PoolSize
+	underWaterAlert["actions"] = []string{
+		"provision additional hosts;",
+		"increase maximum pool size;",
+		"reduce workload;",
+		"deactivate tasks;",
+	}
+	grip.AlertWhen(time.Duration(distroData.totalTasksDuration/float64(distro.PoolSize)) > 24*time.Hour,
```

Comment:
Why not have a single alert for queue duration per host > 24 hours? It doesn't make sense to me to have different behavior depending on if we're at max hosts or not

Reply:
the if is not "are we at max hosts" but rather are we static/not static (distro.PoolSize is 0 for statics.)